storage/innobase/api/api0api.cc

```diff
@@ -1432,7 +1432,7 @@ ib_err_t
 ib_cursor_insert_row(
 /*=================*/
 	ib_crsr_t	ib_crsr,	/*!< in/out: InnoDB cursor instance */
-	const ib_tpl_t	ib_tpl)		/*!< in: tuple to insert */
+	/*const*/ ib_tpl_t	ib_tpl)		/*!< in: tuple to insert */
```

Comment:
This change is problematic as it changes InnoDB internal API (used e.g. on memcached ). Is this necessary ?